rllib/trainer/pg/__init__.py

```diff
@@ -0,0 +1,14 @@
+from ray.rllib.trainer.pg.pg_trainer import PGTrainer, DEFAULT_CONFIG
```

Comment:
I guess we can avoid having to export custom loss functions if we keep trainer and policy overrides in existing directory structure?
see the other comment below.

Reply:
Undid the moving of PG into `rllib/trainer`. We can do this another time and maybe all Trainer classes at once.